pages/business-water-supply.js

```diff
@@ -31,13 +31,6 @@ const WaterSupply = (props) => (
           >
             {props.t("What type of water supply does this establishment use?")}
           </Fieldset.Legend>
-          <HiddenTextAccessible summary={props.t("What is an establishment?")}>
```

Comment:
Did you double check these deletions with Simon?
FSA may not be happy with removing these hint texts entirely, changing them to be more relevant might be the preferred option.

Reply:
I agree with Josh.  Personally I think most of these are fine left in because the question asks something related to the 'establishment', so this explanation is to make clear what the 'establishment' refers to.